portal/src/components/roles-and-groups/list/common/ActionButtonCell.tsx

```diff
@@ -4,26 +4,36 @@ import ActionButton from "../../../../ActionButton";
 import styles from "./ActionButtonCell.module.css";
 import { Text } from "@fluentui/react";
 import { useSystemConfig } from "../../../../context/SystemConfigContext";
+import cn from "classnames";
 
 interface ActionButtonCellProps {
   text: string;
   onClick?: (e: any) => void;
+  disabled?: boolean;
 }
 
 function ActionButtonCell(props: ActionButtonCellProps): React.ReactElement {
-  const { text, onClick } = props;
+  const { text, onClick, disabled } = props;
   const { themes } = useSystemConfig();
   return (
     <BaseCell>
       <ActionButton
         text={
-          <Text className={styles.actionButtonText} theme={themes.destructive}>
+          <Text
+            // TO BE CONFIRMED: How the themes color work for different states
+            className={cn(
+              styles.actionButtonText,
+              disabled && "text-[#BFBFC3]"
+            )}
+            theme={themes.destructive}
+          >
```

Comment:
@tung2744 Not so sure the correct way to set the disabled color here. When to use this themes and when to not use it?

Reply:
It seems it should be
```
      <ActionButton
        text={text}
        styles={{
          label: { fontWeight: 600 },
          labelDisabled: { color: "#BFBFC3" },
        }}
        className={styles.actionButton}
        theme={themes.destructive}
        onClick={onClick}
        disabled={disabled}
      />
```